final_paper_cr_lao

CamSalda opened the pull request:
The end is near...

This PR includes all the changes and materials for the paper version that will be sent for publishing.

CamSalda committed:
adding diagrams

CamSalda committed:
setting the notebook for descriptive statistics

CamSalda committed:
adding EDA stuff

CamSalda committed:
geo spatial eda

CamSalda committed:
reordering files

CamSalda committed:
solving pre-commit issue

CamSalda committed:
adjusting plot files

CamSalda committed:
solving sampling of the frequency model

CamSalda committed:
solving rebase conflicts

CamSalda committed:
adding progress

CamSalda committed:
update shapefile data loaders

CamSalda committed:
final frequency model version

CamSalda committed:
fixing damages sampling

CamSalda committed:
paper progress

CamSalda committed:
final time series notebooks

CamSalda committed:
final time series notebooks

CamSalda committed:
damages curves

CamSalda committed:
HSGP results

CamSalda committed:
final sea notebook

CamSalda committed:
final notebook question

CamSalda committed:
precommit adjust

CamSalda committed:
deleting un-used demos

CamSalda commented:
Hey @jessegrabowski do you thing we can merge this one? It contains all the stuff for the final paper.

jessegrabowski reviewed (changes requested):
Looks fine, but there's no need for the archive folder, just delete the files. They can still be recovered by going back to a previous commit if we ever need them (anything that was on git at any point lives forever)

CamSalda committed:
deleting archive

CamSalda commented:
> Looks fine, but there's no need for the archive folder, just delete the files. They can still be recovered by going back to a previous commit if we ever need them (anything that was on git at any point lives forever)

@jessegrabowski Yeah you are right. I still have to master those "time travel" magic tricks. Every time I try it I end up with a mess. That is why I added the archive.

Deleted.
